web/src/components/questions/UnsupportedAutoYaST.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) [2025] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import { Flex, Grid, GridItem, Text } from "@patternfly/react-core";
+import { AnswerCallback, Question } from "~/types/questions";
+import { Page, Popup } from "~/components/core";
+import { _ } from "~/i18n";
+import QuestionActions from "~/components/questions/QuestionActions";
+import { sprintf } from "sprintf-js";
+
+const UnsupportedElements = ({
+  elements,
+  title,
+  description,
+}: {
+  elements: string[];
+  title: string;
+  description: string;
+}) => {
+  if (elements.length === 0) {
+    return undefined;
+  }
+
+  return (
+    <GridItem sm={12} lg={6}>
+      <Page.Section title={title} description={description}>
+        <Flex gap={{ default: "gapSm" }} aria-role="list">
+          {elements.map((e: string, i: number) => (
+            <span key={i} aria-role="listitem">
+              {e}
+            </span>
+          ))}
+        </Flex>
+      </Page.Section>
+    </GridItem>
+  );
+};
+
+export default function UnsupportedAutoYaST({
+  question,
+  answerCallback,
+}: {
+  question: Question;
+  answerCallback: AnswerCallback;
+}) {
+  const actionCallback = (option: string) => {
+    question.answer = option;
+    answerCallback(question);
+  };
+
+  const planned = question.data.planned?.split(",") || [];
+  const unsupported = question.data.unsupported?.split(",") || [];
+
+  return (
+    <Popup isOpen title={_("Not supported AutoYaST elements")}>
```

Comment:
```suggestion
    <Popup isOpen title={_("Unsupported AutoYaST elements")}>
```

"not supported" works well in a predicate position like "Foo is not supported" and below,
 but not so well in an adjective position like above.